.changeset/early-mayflies-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'slate': patch
+---
+
+add interface shouldMergeNodesRemovePrevNode
```

Comment:
```suggestion
---
'slate': minor
---

Add a `shouldMergeNodesRemovePrevNode` editor method to control when `Transforms.mergeNodes` should remove the previous node rather than carrying out a merge operation. 
```